src/pyxplor/plot_binary.py

```diff
@@ -47,13 +56,149 @@ def plot_binary(
+        raise ValueError("label_fontsize must be a number (integers or floats).")
+
+    # Check if plot_orientation is either 'h' or 'v'
+    valid_plot_orientations = {'h', 'v'}
+    if plot_orientation not in valid_plot_orientations:
+        raise ValueError("Invalid value for 'plot_orientation'. It should be either 'h' or 'v'.")   
+
+    # Check figsize is a tuple of 2 numbers
+    if not (isinstance(figsize, tuple) and len(figsize) == 2 and
+            isinstance(figsize[0], (int, float)) and isinstance(figsize[1], (int, float))):
+        raise ValueError("figsize must be a tuple of exactly two numbers (integers or floats).")
+            
+    # Check whether output is a boolean value
+    if not isinstance(output, bool):
+        raise ValueError("Output must be a boolean value.")
+
+    # Check whether super_title is a string
+    if not isinstance(super_title, str):
+        raise ValueError("super_title must be a string.")
+
+    # Check whether super_title_font is a number
+    if not isinstance(super_title_font, (int, float)):
+        raise ValueError("super_title_font must be a number (integer or float).")
+
+            
+    # Calculate dimension of subplot
+    total_plots = len(list_of_variables)
+    rows = math.ceil(math.sqrt(total_plots))
+    cols = math.ceil(total_plots / rows)
+
+    # Instantiate fig and ax
+    fig, ax = plt.subplots(rows, cols, figsize=figsize)
+    ax = ax.flatten()
+
+    # Plot the variables
+    for i, variable in enumerate(list_of_variables):
+
+        # Countplot
+        if plot_kind == "count":
+            
+            # Horizontal Orientation
+            if plot_orientation == "h":
+                sns.countplot(ax=ax[i], y=variable, data=input_df)
+
+                # Add horizontal labels
+                for p in ax[i].patches:
+                    ax[i].annotate(
+                        int(p.get_width()),
+                        (p.get_width(), p.get_y() + p.get_height() / 2.),
+                        ha = 'center',
+                        va = 'center',
+                        xytext = (label_offset, 0),
+                        textcoords = 'offset points',
+                        fontsize = label_fontsize
+                    )
+            
+            # Vertical Orientation
+            else:
+                sns.countplot(ax=ax[i], x=variable, data=input_df)
+
+                # Add vertical labels
+                for p in ax[i].patches:
+                    ax[i].annotate(
+                        int(p.get_height()),
+                        (p.get_x() + p.get_width() / 2., p.get_height()),
+                        ha = 'center',
+                        va = 'center',
+                        xytext = (0, label_offset),
+                        textcoords = 'offset points',
+                        fontsize = label_fontsize
+                    )
+
+            # Add subplot titles
+            ax[i].set_title('Distribution of {}'.format(variable))
```

Comment:
We should decide a consistent format for this - @iris0614 did the following:

```suggestion
            ax[i].set_title(variable)
```